src/jit/codegenxarch.cpp

```diff
@@ -263,10 +263,12 @@ BasicBlock* CodeGen::genCallFinally(BasicBlock* block, BasicBlock* lblk)
     }
     else
     {
+#ifndef JIT32_GCENCODER
         // Because of the way the flowgraph is connected, the liveness info for this one instruction
```

Comment:
Please add a "TODO" comment to remind us to look at this, e.g.:
```
// TODO-Linux-x86: Do we need to handle the GC information for this NOP or JMP specially, as is done for other architectures?
```

---
In reply to: [95926754](https://github.com/dotnet/coreclr/pull/8889#discussion_r95926754) [](ancestors = 95926754)